src/Components/Layout/AppLayout.css

```diff
@@ -1,5 +1,5 @@
 .layout {
-  display: grid;
+  display: grid !important;
```

Comment:
avoid using !important in css if possible.

Reply:
i have used `ant design` and they use inline css sometimes. to override that i did this! Though try to find some other way too.